src/shared/form/RadioList.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Control, Errors } from 'react-redux-form';
+
+const RadioList = (props) => {
+  const {
+    id,
+    label,
+    name,
+    options,
+    model,
+    messages,
+    validators
+  } = props;
+  return (
+    <div className="field">
+      <fieldset>
+        <legend>{label}</legend>
+        <Errors
+          model={model}
+          show="touched"
+          messages={messages}
+          wrapper={(props) => (
+            <a className="validation-message" href={`#${id}`}><span className="visuallyhidden">Validation Error: </span>{props.children}</a>
+          )}
+        />
+        {options.map((option, i) => {
+          let fieldId = `${id}-${option.value}`;
+          return (
+            <div key={i}>
+              <Control.radio
+                model={model}
+                name={name}
+                id={fieldId}
+                value={option.value}
+                validators={validators}
+              />
+              <label htmlFor={fieldId}>{option.label}</label>
```

Comment:
You could wrap the control in label here to make the label (explicit) rather than have it aside (implicit). This might improve tabbing, highlight or clicking.